### ISSUE TITLE
Hide the voucher screen if there are no vouchers

### ISSUE THREAD

**Issue by jamalv** [teamforus/me#63]
**Is your feature request related to a problem? Please describe.**
The core user group of the me app for now in Zuidhorn will be shopkeepers. They will not have vouchers and will primarily use the QR code scanner.

<img src="https://user-images.githubusercontent.com/30194799/47257586-20794e00-d490-11e8-94b2-fa34be3a1bc5.png" width="20%" height="20%">

**Describe the solution you'd like**
Hide the voucher tab if there are no vouchers. So then there will only be two tabs (QR-code scanner and Profiel)

**Comment by maxvisser** [comment]
I see two options: hiding it or embracing it.
Embracing means it needs functionality for provider: maybe add some information on how to get a voucher?

Or at the funds where the provider is connected to and show transactions per fund.

**Comment by jamalv** [comment]
Agreed a shopkeeper app will make sense at some point, the me app and forus will need more traction before that though. :)

Until then; it can be hidden by default and periodically check if there are any vouchers. If there are, the userdefaults (local storage) can be updated, from that moment on it doesn’t need to check anymore.

I agree with the design choice; that’s also why I created: 
https://github.com/teamforus/me/issues/64

For shopkeepers it is very practical if the me app opens on the qr code scanner, since that is the main functionality they use, and it is what they are currently used to.

**Comment by maxvisser** [comment]
I think this issue has currently the least priority. Me  'for business' would make sense indeed. Therefore it would make sense for now to keep the voucher screen as-is.

To fix this issue we can make a strategy for such a shopkeeper app to exist. I am personally not a big fan of doing that now as we already have a hard time fixing two native apps ;)

**Comment by jamalv** [comment]
Yea, that's not a priority at all. We can be mindful of it, but it's very premature now.

I still am a proponent of the proposal in the previous comment, don't think its a big effort and would make the app more familiar for current shopkeepers (main user group of the me app) and more practical for new ones.

Shopkeepers are the only ones forced to open the me app. Landing on a voucher screen they cannot use could be confusing. Also, every time they open the app (often) they have to switch tabs.

**Comment by maxvisser** [comment]
Jamal could you spec this out? Personally I feel it ok right now.

**Comment by jamalv** [comment]
updated the issue, would like it to be added to jan 1. milestone.

**Comment by jknnr** [comment]
Guys I think we can stick to this idea: toggle switch in the profile that says "start the app from qr-code scanner". So the shopkeepers will not even interact with the voucher screen.
Please like this comment if I can implement this. @jamalv @maxvisser

**Comment by jknnr** [comment]
Update. Currently the toggle switch is implemented, later on we can hide the tab, but more likely we will find another useful content for the shopkeeper and place it there. I think this issue can be closed and later on we will create a new one

**Comment by jamalv** [comment]
I still think hiding the tab is the supreme solution here :)

https://github.com/teamforus/me/issues/172#issuecomment-442141424 is a great suggestion, but it still would not be shown for all shopkeepers. Therefore, I think hiding the tab by default will still be a solid UX improvement for the shopkeepers that are simply using the app as a register for payments.
